src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -178,6 +179,7 @@ subroutine mixedlayer_restrat_general(h, uhtr, vhtr, tv, forces, dt, MLD_in, Var
   real, dimension(SZI_(G)) :: dK, dKm1 ! Depths of layer centers [H ~> m or kg m-2].
   real, dimension(SZI_(G)) :: pRef_MLD ! A reference pressure for calculating the mixed layer
                                        ! densities [R L2 T-2 ~> Pa].
+  real, dimension(SZI_(G)) :: covTS, varS !SGS TS covariance, S variance in Stanley param; currently 0
```

Comment:
Please describe the units of all dimensional real variables in the comments accompanying their declaration statements.  In this case, I believe that the units of covTS should be `[degC ppt]`, while that of varS should be `[ppt2]`.